jobs/build-charms/charms.py

```diff
@@ -269,18 +270,24 @@ def info(name, **query):
 
     def status(self, charm_entity):
         """Read CLI Table output from charmcraft status and parse."""
-        charm_status = self.charmcraft.status(charm_entity)
+        charm_status = self.charmcraft.status(charm_entity, _out=None)
```

Comment:
this silences the output of `charmcraft status <charm>` since it's obnoxious how big it is and will soon be replaced with a new version of this command where we parse the json format.